lib/common/CommonMin.h

```diff
@@ -49,9 +54,8 @@ using namespace Memory;
 #include "DataStructures/SList.h"
 #include "DataStructures/Dlist.h"
 #include "DataStructures/KeyValuePair.h"
-#include "DataStructures/DictionaryEntry.h"
-#include "DataStructures/DefaultContainerLockPolicy.h"
```

Comment:
Do you recall why you need to move ```HeapConstants.h```, ```DefaultContainerLockPolicy.h``` out of this header and make the changes in this file?